File: Sortings/insertionSort.java
package Sortings;

public class insertionSort {
    public static void insertionSort(int[] arr){
        int n = arr.length;
        for(int i = 0; i < n; i++){
            int j = i;
            while( j > 0 && arr[j - 1] > arr[j]){
                int temp = arr[j - 1];
                arr[j - 1] = arr[j];
                arr[j] = temp;
                j--;
            }
        }
        System.out.println("Sorted array: ");
        for(int i = 0; i < n; i++){
            System.out.print(arr[i] + " ");
        }
    }
    public static void main(String[] args) {
        int[] arr = {12, 11, 13, 5, 6};
        insertionSort(arr);
    }
}

File: Sortings/insertionSort.py
def insertionSort(arr):
    n = len(arr)
    for i in range(0, n):
        j = i
        while j > 0 and arr[j - 1] > arr[j]:
            temp = arr[j - 1]
            arr[j - 1] = arr[j]
            arr[j] = temp
            j= j - 1
            
    return arr

# Example usage
if __name__ == "__main__":
    arr = [12, 11, 13, 5, 6]
    print("Unsorted array:", arr)
    sorted_arr = insertionSort(arr)
    print("Sorted array:", sorted_arr)